integration/presto/src/main/prestosql/org/apache/carbondata/presto/PrestoFilterUtil.java

```diff
@@ -78,16 +78,22 @@ private static DataType spi2CarbondataTypeMapper(HiveColumnHandle columnHandle)
     HiveType colType = columnHandle.getHiveType();
     if (colType.equals(HiveType.HIVE_BOOLEAN)) {
       return DataTypes.BOOLEAN;
+    } else if (colType.equals(HiveType.HIVE_BINARY)) {
+      return DataTypes.BINARY;
     } else if (colType.equals(HiveType.HIVE_SHORT)) {
       return DataTypes.SHORT;
     } else if (colType.equals(HiveType.HIVE_INT)) {
       return DataTypes.INT;
+    } else if (colType.equals(HiveType.HIVE_FLOAT)) {
```

Comment:
please handle the same file changes in prestodb also and compile prestodb profile once with latest changes

Reply:
also manually run new testcases once in prestodb profile

Reply:
done